cli/tests/data/marathon/pods/updated_good.json

```diff
@@ -15,5 +15,10 @@
         "mem": 16.0
       }
     }
+  ],
+  "networks": [
```

Comment:
Yeah, the integration tests failed because they expected a 200 status but got a 422.